### DISCUSSION TITLE
Consolidate #dev-frontend and #dev-backend into a single channel

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1183] [Suggested Changes] [answered]
It seems almost every time I want to post a #dev-related thing, it doesn't fit neatly into #dev-frontend or #dev-backend. So I wind up just posting it in #random.

This time it was a free JSON API for testing. A previous example was a CORS-related thing.

In a VC event, Kirk made a comment that the line between frontend and backend is blurring. I think that's at play here.

**Comment by BekahHW** [comment, marked as answer]
Why not post in #dev-fullstack?

**Reply by ebanner** [reply]
TIL about #dev-fullstack 😅🤯
